chore(android): update play store data for video/photo permissions

The play store has a new metadata requirement around video/photo permissions. We don't use these so we need to mark that accordingly (I haven't looked into how, yet).

https://build.palaso.org/buildConfiguration/KeymanAndroid_Build/518310?buildTab=log&linesState=2421&logView=flowAware&focusLine=2820

```
01:18:39   FAILURE: Build failed with an exception.
01:18:39   Committing changes
01:18:39   
01:18:39   * What went wrong:
01:18:39   Execution failed for task ':commitEditForComDotTavultesoftDotKmapro'.
01:18:39   > A failure occurred while executing com.github.triplet.gradle.play.tasks.CommitEdit$Committer
01:18:39      > 403 Forbidden
01:18:39        POST https://androidpublisher.googleapis.com/androidpublisher/v3/applications/com.tavultesoft.kmapro/edits/04516564361834714869:commit?changesNotSentForReview=false
01:18:39        {
01:18:39          "code": 403,
01:18:39          "errors": [
01:18:39            {
01:18:39              "domain": "global",
01:18:39              "message": "You must let us know whether your app uses any photo and video permissions.",
01:18:39              "reason": "forbidden"
01:18:39            }
01:18:39          ],
01:18:39          "message": "You must let us know whether your app uses any photo and video permissions.",
01:18:39          "status": "PERMISSION_DENIED"
01:18:39        }
01:18:39   
```

--- a/android/KMAPro/kMAPro/src/main/AndroidManifest.xml
+++ b/android/KMAPro/kMAPro/src/main/AndroidManifest.xml
@@ -7,10 +7,6 @@
   <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
   <uses-permission android:name="android.permission.VIBRATE" />
 
-  <!-- API 33 and above -->
-  <uses-permission android:name="android.permission.READ_MEDIA_IMAGES" />
-  <uses-permission android:name="android.permission.READ_MEDIA_VIDEO" />
-
   <!-- Devices running up to Android 12L
        Starting in API level 33, this permission has no effect:
        https://developer.android.com/reference/android/Manifest.permission#READ_EXTERNAL_STORAGE -->

--- a/android/KMAPro/kMAPro/src/main/java/com/tavultesoft/kmapro/CheckPermissions.java
+++ b/android/KMAPro/kMAPro/src/main/java/com/tavultesoft/kmapro/CheckPermissions.java
@@ -35,8 +35,8 @@ public class CheckPermissions {
         checkPermission(activity, Manifest.permission.READ_EXTERNAL_STORAGE);
     } else {
       // API 33+
-      //https://developer.android.com/about/versions/13/behavior-changes-13#granular-media-permissions
-      // Manifest.permission.READ_MEDIA_AUDIO doesn't seem to be needed
+      // We had to remove these MEDIA permissions from AndroidManifest.xml so these will end up failing
+      // https://support.google.com/googleplay/android-developer/answer/14115180?hl=en
       permissionsOK = permissionsOK && checkPermission(activity, Manifest.permission.READ_MEDIA_IMAGES);
       permissionsOK = permissionsOK && checkPermission(activity, Manifest.permission.READ_MEDIA_VIDEO);
     }

--- a/android/docs/help/troubleshooting/grant-storage-permission.md
+++ b/android/docs/help/troubleshooting/grant-storage-permission.md
@@ -19,13 +19,7 @@ permission to access storage / "file and media". The screenshot below is from An
 
 ### Android 13.0+ Devices
 
-Step 1)
-Go to Android Settings.
+Due to Google Play Store's "Photo and Video Permissions policy", 
+Keyman for Android must use a system picker to browse for .kmp files from within the Keyman app.
 
-Step 2)
-Depending on your device, click "Apps" and grant Keyman permission to the following:
-  * Photos and videos --> Always allow all
-
-The screenshot below is from Android 14.0.
-
-![](../android_images/keyman-storage-permission-34b.png)
+Follow the steps in [Installing Custom Keyboards/Dictionaries](../basic/installing-custom-packages).

